src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -863,7 +854,6 @@ EnsureRelationCanBeDistributed(Oid relationId, Var *distributionColumn,
 
 	EnsureTableNotDistributed(relationId);
 	EnsureLocalTableEmptyIfNecessary(relationId, distributionMethod, viaDeprecatedAPI);
-	EnsureReplicationSettings(InvalidOid, replicationModel);
```

Comment:
I needed to remove this check because the function no longer makes sense here. The user can not set the replication model via a GUC

Reply:
is the other call still useful?

Reply:
No, not really. Removed it as well after some testing